### PR TITLE
Update macros.html

### PR DESCRIPTION
**Pull request by Jieiku**
Its a Zola theme, I fixed the footer for you to reflect this. (I know you originally ported it from jekyll and probably just missed this.)

### CODE REVIEW

**Comment by Jieiku** [comment]
I also just added a commit for screenshot.png which is a requirement for this theme to be included here: https://www.getzola.org/themes/

File: templates/macros.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 {% macro footer() %}
 <footer class="footer">
     <p>&copy; {{ config.extra.username }}</p>
-    <p>Build with Jekyll and <span class="love">❤</span> by <a href="https://github.com/nrandecker">Nathan Randecker</a></p>
+    <p>Built with Zola and <span class="love">❤</span> by <a href="https://github.com/nrandecker">Nathan Randecker</a></p>
```
Comment (svavs) [inline templates/macros.html]: Maybe the following would be better ;)
```html
<p>Ported to Zola with <span class="love">❤</span> by <a href="https://github.com/svavs">Silvano Sallese</a> (original by <a href="https://github.com/nrandecker">Nathan Randecker</a>)</p>
```

**Review by svavs — APPROVED** [review]
Thank you very much, in this theme I carelessly saved the screenshot with the wrong extension.

**Comment by Jieiku** [comment]
no problem, its a good looking theme!

File: templates/macros.html (hunk 1 above)
Comment (Jieiku) [reply templates/macros.html]: yeah, looks great!

**Comment by Jieiku** [comment]
I updated the pull request to have the new footer text you suggested.

**Comment by Jieiku** [comment]
Let me know if you see anything else you want updated? Otherwise I think it is ready for you to merge.

**Comment by svavs** [comment]
> no problem, its a good looking theme!

yes, very effective theme indeed.

Thank you for your contribution. Very appreciated.

**Review by svavs — COMMENTED** [review]
ready to merge. thank you.

**Pull request merged**